### PR TITLE
for #7236 Add metrics tags for all issues labels

### PR DESCRIPTION
**Pull request by iorgamgabriel**
No tests , Screenshots and Accessibility needed 

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on `Show All Checks`,
2. click `Details` next to `build-focus-debug` or `build-klar-debug` for changes targeting Klar,
2. click `View task in Taskcluster`,
3. click the `Artifacts` row,
4. click to download any of the apks listed here which use an appropriate name for each CPU architecture.

### CODE REVIEW

**Comment by Mugurell** [comment]
Found it strange that running the script only adds 1 tag but it seems like Focus does not yet have feature labels as Fenix has - https://github.com/mozilla-mobile/fenix/labels?q=feature

**Comment by iorgamgabriel** [comment]
I added all the almost all the tags from Fenix

File: app/tags.yaml [hunk 1]
```diff
@@ -2,12 +2,174 @@
...
+Voice:
+  description: Corresponds to the [Feature:Voice](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3AVoice)
+    label on GitHub.
+WebExtensions:
+  description: Corresponds to the [Feature:WebExtensions](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3AWebExtensions)
+    label on GitHub.
 performance:
   description: Used for data reviews to label metrics related to performance. Corresponds
-    to the [Feature:Performance](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3APerformance)
+    to the [Feature:performance](https://github.com/mozilla-mobile/focus-android/issues?q=label%3AFeature%3Aperformance)
```
Comment (Mugurell) [inline app/tags.yaml:174]: Seeing that all other tags use an uppercase can we rename this tag on Github to also use uppercase?
(Then the script will have to be run again to ensure `Performance` remain as a tag)

**Review by Mugurell — APPROVED** [review]
Looks like adding telemetry on Focus will get a lot easier. 🎉

File: app/tags.yaml (hunk 1 above)
Comment (iorgamgabriel) [reply app/tags.yaml]: Done

**Pull request merged**